pkg/aws/client.go

```diff
@@ -431,10 +432,24 @@ func (c *awsClient) GetRegion() string {
 	return aws.StringValue(c.awsSession.Config.Region)
 }
 
-func (c *awsClient) GetSubnetIDs() ([]*ec2.Subnet, error) {
+func (c *awsClient) ListSubnets() ([]*ec2.Subnet, error) {
 	return c.getSubnetIDs(&ec2.DescribeSubnetsInput{})
 }
 
+func (c *awsClient) GetSubnetsById(subnetIds []string) ([]*ec2.Subnet, error) {
```

Comment:
I feel like these two are essentially the same and we could probably have a single function that receives one or more subnets with the spread operator. What do you think?

Reply:
Yeah that's a good idea @gdbranco . I'll make the changes.